src/services/hotels.js

```diff
@@ -0,0 +1,47 @@
+import models from '../database/models';
+
+const { hotels } = models;
+
+class HotelServices {
+  static createHotel(newHotel) {
+    return hotels.create(newHotel);
+  }
+
+  static findByProp(prop) {
+    return hotels.findAll({
+      where: prop,
+    });
+  }
+
+  static findOneHote(prop) {
+    return hotels.findOne({
+      where: {
+        id: 1,
+      },
+      include: ['hotelsCity']
+    });
+  }
+
+  // static getUsers() {
+  //   return users.findAll();
```

Comment:
Please consider removing files you inserted which you are no longer using and consider also removing unnecessary comments

Reply:
Sure!